buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeImplPlugin.kt

```diff
@@ -390,7 +390,13 @@ private fun configureComposeCompilerPlugin(
 
             // Gradle hack ahead, we use of absolute paths, but is OK here because we do it in
             // doFirst which happens after Gradle task input snapshotting. AGP does the same.
-            compile.doFirst {
+
```

Comment:
I think, we shouldn't squash this PR, and preserve commits which were cherry-picked. We have 2 ways:

1. cherry-pick them to jb-main, and just push it without PR
2. allow rebase PR-merging

What do you think is better?

If we go p.2, better to mention which commits were cherry-picked, to make review easier (so we don't look at things that are already reviewed in AOSP)

Reply:
p1 should be easier for review I think. I'll update the PR.